InternalPythonModules/android/browserlocation.py

```
@@ -109,15 +114,18 @@ def __findGeoLocationsInDB(self, databasePath, abstractFile):
                     self._logger.log(Level.SEVERE, traceback.format_exc())
                     MessageNotifyUtil.Notify.error("Failed to index GPS trackpoint artifact for keyword search.", artifact.getDisplayName())
 
+        except SQLException as ex:
+            # Unable to execute browser location SQL query against database.
+            pass
         except Exception as ex:
-            self._logger.log(Level.SEVERE, "Error putting artifacts to blackboard", ex)
-            self._logger.log(Level.SEVERE, traceback.format_exc())
+            # Error putting artifacts to blackboard.
```

Why are we swallowing this exception?